docs/node-port-local.md

```diff
@@ -137,6 +138,7 @@ annotation is deprecated. The array contains a single member, equal to the
 `protocol` field.
 The `protocols` field will be removed from Antrea for minor releases post March 2023,
 as per our deprecation policy.
+Starting from Antrea v2.0, the `protocols` field is removed.
```

Comment:
I feel the whole content starting with "Starting from the Antrea v1.7 minor release, the `protocols` field ..." can be removed. There seems no much value to detail the history of a removed field when there is already a "Usage pre Antrea v1.7" section.

Reply:
Done